src/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Manager.cpp

```diff
@@ -468,16 +443,19 @@ const CSSISpaceWeather* Manager::accessCSSISpaceWeatherAt(const Instant& anInsta
 
     if (mode_ == Manager::Mode::Automatic)
     {
-        const Instant currentInstant = Instant::Now();
+        ManifestManager& manifestManager = ManifestManager::Get();
+
+        const Instant CSSIManifestUpdateTimestamp =
+            manifestManager.getLastUpdateTimestampFor(CSSISpaceWeatherManifestName);
 
         if ((!CSSISpaceWeatherUpdateTimestamp_.isDefined()) ||
-            ((CSSISpaceWeatherUpdateTimestamp_ + Duration::Weeks(1.0)) < currentInstant))  // [TBM] Param
+            (CSSISpaceWeatherUpdateTimestamp_ < CSSIManifestUpdateTimestamp))  // [TBM] Param
```

Comment:
```suggestion
            (CSSISpaceWeatherUpdateTimestamp_ < CSSIManifestUpdateTimestamp))
```
Does the [TBM] Param still need to be there?